Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.h

```diff
@@ -0,0 +1,16 @@
+//
+//  ADTestAppProfileViewController.h
+//  MyTestiOSApp
+//
+//  Created by Ryan Pangrle on 4/29/16.
+//  Copyright © 2016 MS. All rights reserved.
+//
```

Comment:
Put correct headers. Can  you check if we missed any other files too in old CRs?